nucypher/cli/literature.py

```diff
@@ -114,19 +114,38 @@
 
 NO_STAKES_FOUND = "No stakes found."
 
+
 POST_STAKING_ADVICE = """
 View your stakes by running 'nucypher stake list'
 or set your Ursula worker node address by running 'nucypher stake bond-worker'.
 
 See https://docs.nucypher.com/en/latest/staking/running_a_worker.html
 """
 
-
 #
 # Events
 #
+
 CONFIRM_OVERWRITE_EVENTS_CSV_FILE = "Overwrite existing CSV events file - {csv_file}?"
 
+#
+# Remove Unused
+#
+
+
+FETCHING_INACTIVE_STAKES = 'Fetching inactive stakes'
+
+NO_INACTIVE_STAKES = "No inactive stakes found\n"
+
+CONFIRM_REMOVE_ALL_UNUSED_SUBSTAKES = """
+This action will perform a series of transactions to remove all unused sub-stakes
+(Indices {stakes}).  Verify each transaction is successful while proceeding.
+
+https://etherscan.io/tx/{staker_address}
```

Comment:
What about combining these lines...? Also, did you mean `address` for the URL instead of `tx`?

```suggestion
(Indices {stakes}).  It is recommended that you verify each staker transaction was successful (https://etherscan.io/address/{staker_address}).
```